v2/api/resources/v1beta20200601/resourcegroup_types.go

```diff
@@ -0,0 +1,344 @@
+func (rg *ResourceGroup) GetAPIVersion() string {
+	return "2020-06-01"
+}
+
+// GetType returns the ARM Type of the resource. This is always "Microsoft.Resources/resourceGroups"
+func (rg *ResourceGroup) GetType() string { return "Microsoft.Resources/resourceGroups" }
+
+// SetStatus sets the status of this resource
+func (rg *ResourceGroup) SetStatus(status genruntime.ConvertibleStatus) error {
+	// If we have exactly the right type of status, assign it
+	if st, ok := status.(*ResourceGroupStatus); ok {
+		rg.Status = *st
+		return nil
+	}
+
+	// Convert status to required version
+	var st ResourceGroupStatus
+	err := status.ConvertStatusTo(&st)
+	if err != nil {
+		return errors.Wrap(err, "failed to convert status")
+	}
+
+	rg.Status = st
+	return nil
+}
+
+// GetResourceKind returns the kind of the resource
+func (rg *ResourceGroup) GetResourceKind() genruntime.ResourceKind {
+	return genruntime.ResourceKindNormal
+}
+
+var _ genruntime.LocatableResource = &ResourceGroup{}
+
+func (rg *ResourceGroup) Location() string {
+	if rg.Spec.Location == nil {
+		return ""
+	}
+	return *rg.Spec.Location
+}
+
+var _ conversion.Hub = &ResourceGroup{}
+
+// Hub marks that this ResourceGroup is the hub type for conversion
+func (rg *ResourceGroup) Hub() {}
+
+// +kubebuilder:object:root=true
+type ResourceGroupList struct {
+	metav1.TypeMeta `json:",inline"`
+	metav1.ListMeta `json:"metadata,omitempty"`
+	Items           []ResourceGroup `json:"items"`
+}
+
+type ResourceGroupStatus struct {
+	ID *string `json:"id,omitempty"`
+
+	Name     *string `json:"name,omitempty"`
+	Location *string `json:"location,omitempty"`
+
+	// ManagedBy is the management group responsible for managing this group
+	ManagedBy *string `json:"managedBy,omitempty"`
+
+	// Tags are user defined key value pairs
+	Tags map[string]string `json:"tags,omitempty"`
+
+	ProvisioningState *string `json:"provisioningState,omitempty"`
+
+	// Conditions describe the observed state of the resource
+	Conditions []conditions.Condition `json:"conditions,omitempty"`
+}
+
+var (
+	_ genruntime.FromARMConverter  = &ResourceGroupStatus{}
+	_ genruntime.ConvertibleStatus = &ResourceGroupStatus{}
+)
+
+func (status *ResourceGroupStatus) NewEmptyARMValue() genruntime.ARMResourceStatus {
+	return &ResourceGroupStatusARM{}
+}
+
+// PopulateFromARM populates a Kubernetes CRD object from an Azure ARM object
+func (status *ResourceGroupStatus) PopulateFromARM(owner genruntime.ArbitraryOwnerReference, armInput interface{}) error {
+	typedInput, ok := armInput.(ResourceGroupStatusARM)
+	if !ok {
+		return fmt.Errorf("unexpected type supplied for PopulateFromArm() function. Expected ResourceGroupStatusArm, got %T", armInput)
+	}
+	status.ID = typedInput.ID
+	status.Location = typedInput.Location
+	status.ManagedBy = typedInput.ManagedBy
+	status.Name = typedInput.Name
+	status.Tags = typedInput.Tags
+	// Set property ‘AccessTier’:
```

Comment:
I copied the `v1alpha1api` variant of the file and made minimal changes, so I have no idea. 
Since it's not a formal TODO, I think it's debris - will remove from both files.